fawltydeps/extract_declared_dependencies.py

```diff
@@ -5,9 +5,11 @@
 import logging
 import re
 import sys
+from dataclasses import replace
```

Comment:
Nit:

When 2 conditions are true, I especially favor just importing the library and then having use sites be qualified. Those would be...
a) The imported name is a relatively common word / name (`replace`)
b) It's not as typical / standard (i.e., `from functools import partial` violates a, but I'd argue that since it's been around quite some time it's more natural / common.

Reply:
This makes sense, and I would say that my threshold is only slightly lower:
- When you only use one (or very few) names from a module, it is nice to communicate this up front in the import statement.
- Even if the name is a common word, if it is _only_ used as that imported name across the whole file, I don't have a problem with it.
- But if the name introduces _any_ kind of (perceived) ambiguity, e.g. you have a class with a `replace()` method in the same file, then I would definitely argue for using it with qualification everywhere.

In summary (and in this case): grepping for `replace` in the file yields only relevant hits, so I find any extra qualification unnecessary.